tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -403,12 +402,10 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
         {csinfo_.conjugate_transpose,
          mkl_op_registry::GetMklOpName(csinfo_.conjugate_transpose),
          CopyAttrsAll, AlwaysRewrite, kRewriteForOpNameChange});
-#endif  // !ENABLE_MKLDNN_V1
     rinfo_.push_back({csinfo_.conv2d,
                       mkl_op_registry::GetMklOpName(csinfo_.conv2d),
                       CopyAttrsConvCheckConstFilter, AlwaysRewrite,
                       kRewriteForLayoutPropagation});
-#ifndef ENABLE_MKLDNN_V1
```

Comment:
Many of the ops in the removed `#ifndef`s don't seem directly related to fused batch norm or max/avg pool.
Are all these `#ifndef`s removed by mistake?

Reply:
No these are all needed.